resolwe_bio/management/commands/generate_etc.py

```diff
@@ -38,20 +37,24 @@ def create_etc(gene_ids, path, file_name):
         """Generate an expression time course."""
         genes = {}
         with open(os.path.join(path, file_name), 'w') as f:
-            csvwriter = csv.writer(f, delimiter=str('\t'))
-            times = [0, 4, 8, 12, 16, 20, 24]
-            csvwriter.writerow(['Gene'] + times)
-            with gzopen(gene_ids) as gene_ids:
+            # NOTE: Default line terminator is '\r\n'
+            # NOTE: Python2's csv module doesn't accept a unicode string for delimeter
+            csvwriter = csv.writer(f, delimiter=str('\t'), lineterminator='\n')
+            times = (0, 4, 8, 12, 16, 20, 24)
+            csvwriter.writerow(('Gene', ) + times)
+            with gzip.open(gene_ids, mode='rt') as gene_ids:
```

Comment:
@tjanez This breaks when reading gzip files that have alternative line endings (the ouput of the django-admin command is not ok). Since `gzip.open`does not support universal new line endings, we introduced the `resolwe_bio.tools.utils.gzopen`function that is fast, handles different line endings and should be used here if there are no other alternatives.